src/v2/es-utils.js

```diff
@@ -4,20 +4,20 @@
 const fs = require('fs');
 const elasticsearch = require('elasticsearch');
 const nconf = require('nconf');
+const waitOn = require('wait-port');
 const logger = require('../../log.js');
 
 /* eslint no-sync: 0 */
 class ESClass {
+  // Class to handle interactions with elasticsearch
```

Comment:
This is a good candidate for jsdoc-style comments.

Reply:
Absolutely! I'd like to dedicate a round of documentation updates to a future commit/PR - that sound OK?